### PR TITLE
Moved ternary integration tests to bee-test

### PR DESCRIPTION
**Pull request by zesterer**
# Description of change

Moved ternary integration tests to bee-test to allow bee-ternary to be released.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Bug fix (a non-breaking change which fixes an issue)

## How the change has been tested

Literally just ran the tests

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that new and existing unit tests pass locally with my changes
- [x] I have updated the CHANGELOG.md, if my changes are significant enough

### CODE REVIEW

File: bee-test/tests/bee-ternary.rs [hunk 1]
```diff
@@ -0,0 +1 @@
+pub mod ternary;
```
Comment (thibault-martinez) [inline bee-test/tests/bee-ternary.rs]: Header

File: bee-test/tests/ternary/mod.rs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+pub mod buf;
```
Comment (thibault-martinez) [inline bee-test/tests/ternary/mod.rs]: Header

File: bee-test/Cargo.toml [hunk 3]
```diff
@@ -12,7 +12,6 @@ homepage = "https://www.iota.org"
 
 [dependencies]
 bee-ternary = { version = "0.3.2-alpha", features = [ "serde1" ], path = "../bee-ternary" }
-
```
Comment (thibault-martinez) [inline bee-test/Cargo.toml]: Put it back please, for consistency we put a newline between our crates and external crates :)

**Review by thibault-martinez — CHANGES_REQUESTED** [review]

**Review by thibault-martinez — CHANGES_REQUESTED** [review]
Can you change the release date of the last bee-ternary version to today please ? We couldn't release it on Friday

**Review by thibault-martinez — APPROVED** [review]

**Review by pvdrz — APPROVED** [review]

**Review by samuel-rufi — APPROVED** [review]

**Pull request merged**